### PR TITLE
Acknowledge funding agency and contributors in doc

### PR DESCRIPTION
**Pull request by MaxThevenet**
This small PR adds a section in the documentation to acknowledge ECP as well as contributors and users.

### CODE REVIEW

File: Docs/source/acknowledgements.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Acknowledgements
```
Comment (ax3l) [inline Docs/source/acknowledgements.rst]: Maybe even call this "Funding"?

Comment (MaxThevenet) [reply Docs/source/acknowledgements.rst]: I feel like Acknowledgements encompasses both, funding information is found in the acknowledgement section of a paper, so I think "acknowledgement" is general enough, isn't it?

**Comment by dpgrote** [comment]
Here is the standard acknowledgment for LLNL. Could this be added? Note that LBL has a similar statement with its contract number.

> This work was performed in part under the auspices of the U.S. Department of Energy by Lawrence Livermore National Laboratory under Contract DE-AC52-07NA27344.

**Comment by jlvay** [comment]
I think that the acknowledgement to ECP is sufficient. No need to add the individual institution acknowledgements.

File: Docs/source/acknowledgements.rst (hunk 1 above)
Comment (MaxThevenet) [reply Docs/source/acknowledgements.rst]: Renamed it "Funding and Acknowledgements"

**Pull request merged**